src/modlist.cpp

```diff
@@ -337,9 +310,11 @@ QVariant ModList::data(const QModelIndex &modelIndex, int role) const
       }
     }
     else if (column == COL_PRIORITY) {
-      int priority = modInfo->getFixedPriority();
-      if (priority != INT_MIN) {
-        return priority;
+      if (modInfo->isBackup()) {
```

Comment:
Can these calls be put encapsulated somewhere? You're duplicating this code in `COL_PRIORITY` and `PriorityRole` below. Maybe `Profile::getModPriority()` should do that instead? Or add something like `ModInfo::getPriority(Profile&)`? I'm thinking if we ever add another type of mod with fixed prio.

Reply:
The "problem" is that these are completely modlist specific. The profile does not use and care about these "custom" priorities for backups and overwrite so putting something in the profile would be very weird. I'll add a `ModList` member function.